### ISSUE TITLE
Wrong interpretation of language resulting in wrong classification

### ISSUE THREAD

**Issue by ShilpaBhagavath** [facebook/duckling#410]
If the text is  "first Wednesday of next month" and lang =EN; I get correct result.

However if I change the same phrase to French, that is, "Le premier mercredi du mois prochain" with lang = FR,

It gives me results for "Le premier mercredi" (first Wednesday) and the for "mois prochain"(next month) which is wrong. I

Am I doing something wrong or is there anyway to correct this?

(Used google translator to translate to French)

**Comment by patapizza** [comment]
Hi @ShilpaBhagavath,

Seems like we're missing a rule to combine both parts in French.

**Comment by chessai** [comment]
This bug still exists in `master`:
```
> debug (makeLocale FR Nothing) "Le premier mercredi du mois prochain" [Seal Time]
le <time> (Le premier mercredi)
-- regex (Le)
-- intersect (premier mercredi)
-- -- day of month (premier) (premier)
-- -- -- regex (premier)
-- -- Mercredi (mercredi)
-- -- -- regex (mercredi)
<cycle> prochain|suivant|d'après (mois prochain)
-- mois (grain) (mois)
-- -- regex (mois)
-- regex (prochain)
[Entity {dim = "time", body = "Le premier mercredi", value = RVal Time (TimeValue (SimpleValue (InstantValue {vValue = 2013-05-01 00:00:00 -0200, vGrain = Day})) [SimpleValue (InstantValue {vValue = 2013-05-01 00:00:00 -0200, vGrain = Day})] Nothing), start = 0, end = 19, latent = False, enode = Node {nodeRange = Range 0 19, token = Token Time TimeData{latent=False, grain=Day, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 0 2, token = Token RegexMatch (GroupMatch []), children = [], rule = Nothing},Node {nodeRange = Range 3 19, token = Token Time TimeData{latent=False, grain=Day, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 3 10, token = Token Time TimeData{latent=False, grain=Day, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 3 10, token = Token RegexMatch (GroupMatch []), children = [], rule = Nothing}], rule = Just "day of month (premier)"},Node {nodeRange = Range 11 19, token = Token Time TimeData{latent=False, grain=Day, form=Just DayOfWeek, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 11 19, token = Token RegexMatch (GroupMatch ["credi"]), children = [], rule = Nothing}], rule = Just "Mercredi"}], rule = Just "intersect"}], rule = Just "le <time>"}},Entity {dim = "time", body = "mois prochain", value = RVal Time (TimeValue (SimpleValue (InstantValue {vValue = 2013-03-01 00:00:00 -0200, vGrain = Month})) [SimpleValue (InstantValue {vValue = 2013-03-01 00:00:00 -0200, vGrain = Month})] Nothing), start = 23, end = 36, latent = False, enode = Node {nodeRange = Range 23 36, token = Token Time TimeData{latent=False, grain=Month, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 23 27, token = Token TimeGrain Month, children = [Node {nodeRange = Range 23 27, token = Token RegexMatch (GroupMatch []), children = [], rule = Nothing}], rule = Just "mois (grain)"},Node {nodeRange = Range 28 36, token = Token RegexMatch (GroupMatch ["",""]), children = [], rule = Nothing}], rule = Just "<cycle> prochain|suivant|d'apr\232s"}}]
```

**Comment by chessai** [comment]
resolved by ccdf27ad1d34f557d729ec794539ace430fdbdf1
